index.js

```diff
@@ -1,17 +1,39 @@
 var _ = require('lodash');
 
 module.exports = function(data) {
-	var scss = '';
+    var scss = '';
 
     if (!data) {
-    	return;
+        return;
     }
-    
+
     data = JSON.parse(data);
 
     _.mapKeys(data, function(value, key) {
-		scss+= '$' + key + ': ' + value + ';\r\n'
-	});
+        var isObject = typeof(value) === 'object',
+            isArray = Array.isArray(value);
+
+        if (isObject || isArray) {
+            scss+= '$' + key + ': (\n';
+            var i = 0;
+
+            _.mapKeys(value, function(value, key, obj) {
+                i++;
+                var len = Object.keys(obj).length,
+                    comma = i !== len ? ',' : '';
+                if (isArray) {
+                    scss+= '\t' + value + comma + '\r\n';
+                } else {
+                    scss+= '\t' + key + ': ' + value + comma + '\r\n';
+                };
```

Comment:
this could be made recursive so that you can have infinite levels of nesting in the JSON